src/test/java/fr/inria/coming/spoon/core/MainComingTest.java

```diff
@@ -124,6 +124,31 @@ public void testFilePairsMain() throws Exception {
         assertNotNull(result);
     }
 
+
+	@SuppressWarnings("unchecked")
+	@Test
+	public void testOrderOutputopt() throws Exception {
+		ComingMain cm = new ComingMain();
+		Object result = cm.run(new String[] { "-location", "repogit4testv0", "-repairability", "true","-parameters" ,"nb_commit:10"});
+		assertNotNull(result);
+		assertTrue(result instanceof CommitFinalResult);
+		CommitFinalResult cfres = (CommitFinalResult) result;
+		Map<Commit, RevisionResult> commits = cfres.getAllResults();
+
+		List<String> commitsInOrder = new ArrayList<>();
+		for (String commit : this.commitsId) {
+			commitsInOrder.add(commit);
+		}
+
+		int currentIndex = 0;
+		for (Commit commit : commits.keySet()) {
+
```

Comment:
add comment '// checking that we get the first ten ones, as encoded as test data in this class'